docs/personalization/recommendation_models.md

```diff
@@ -211,3 +211,25 @@ The logic used for resolving a submodel is as follows:
     If you do not want to group recommendations based on a certain attribute, 
     remove the attribute parameter from the request. 
     The submodel is then omitted.
+
+## Segments
+
+Segments allow to get personalized content suitable for particular users groups. They compute models based on segment attribute factor.
+Information with user segment are provided in each event which comes from the tracking script.
+
+First, make sure you have [enabled personalization](https://doc.ibexa.co/en/latest/guide/personalization/enabling_personalization/#enable-personalization) and [configured item type tracking](https://doc.ibexa.co/en/latest/guide/personalization/recommendation_client/#set-up-item-type-tracking).
```

Comment:
```suggestion
To use segments, you must first [configure item type tracking](https://doc.ibexa.co/en/latest/guide/personalization/recommendation_client/#set-up-item-type-tracking).
```
Enabling perso seems self-explanatory, we're halfway through an article about perso configuration.